.travis.yml

```diff
@@ -5,6 +5,8 @@ before_install:
 script:
         - ./make.sh
         - make check
+        - cp libcapstone.so bindings/python/
```

Comment:
a simple trick is to replace that line with:

    - cp libcapstone.* bindings/python/

then we can support both Linux & OSX.